dist/mlhr-table.js

```diff
@@ -826,7 +826,7 @@ angular.module('datatorrent.mlhrTable.filters.mlhrTableRowFilter', ['datatorrent
             var col = enabledFilterColumns[i];
             var filter = col.filter;
             var term = searchTerms[col.id];
-            var value = row[col.key];
+            var value = {}.hasOwnProperty.call(options, 'getter') && typeof options.getter === 'function' ? options.getter(col.key, row) : row[col.key];
```

Comment:
go ahead and take out the `typeof options.getter === 'function'` check; it adds unnecessary cycles to this operation. If the user sets a getter and its not a function, it is their fault for not making it a function!